mlir/test/Integration/Dialect/SparseTensor/CPU/sparse_coo_test.mlir

```diff
@@ -171,19 +171,19 @@ module {
                                             -> tensor<8x8xf32>
     %COO_RET = call @add_coo_coo_out_coo(%COO_A, %COO_B) : (tensor<8x8xf32, #SortedCOO>,
                                                             tensor<8x8xf32, #SortedCOOSoA>)
-                                                         -> tensor<8x8xf32, #SortedCOO>
-    %C4 = sparse_tensor.convert %COO_RET : tensor<8x8xf32, #SortedCOO> to tensor<8x8xf32>
```

Comment:
note that here it would have made sense to keep C4 conversion as well, since it tests the conversion, and the printing remains anyway (with count 4 instead of 3)

Reply:
Good point! Will ad it back.